### ISSUE TITLE
Azure Lighthouse

### ISSUE THREAD

**Issue by DeanGross144** [Azure/review-checklists#604]
I can't find any checklists that include Azure Lighthouse. Am I missing something? Are there any plans to add this technology?

**Comment by erjosito** [comment]
Hey @DeanGross144 thanks for the issue! You are raising an excellent point. Where would you see the Lighthouse recommendations? In the Multitenancy checklist probably?

Not answering your question, but as an explanation: the teams that so far have contributed to the checklists are primarily customer-focused, while Lighthouse is AFAIK a partner-focused tech. Consequently, we are not that familiar with it. However, if you have any recommendations that you would like to add, I would be happy to help in the process.

**Comment by DeanGross144** [comment]
Yes, I think it would make sense to have it in the Multitenancy checklist, but it could also show up in the Security checklist, we use lighthouse to access Sentinel for our clients
